src/test/java/uk/gov/justice/digital/zone/structured/StructuredZoneCdcTest.java

```diff
@@ -139,39 +128,21 @@ public void shouldHandleValidDeleteRecords() throws DataStorageException {
 
     @Test
     public void shouldHandleInvalidRecords() throws DataStorageException {
-        givenTheSchemaExists();
         givenTheSourceReferenceIsValid();
 
-        assertNotNull(underTest.process(spark, testDataSet, dataMigrationEventRow));
-    }
-
-    @Test
-    public void shouldHandleNoSchemaFound() throws DataStorageException {
```

Comment:
This test is not valid anymore. The schema has been validated to exists before invoking the methods of the class being tested.